app/views/users/show_white.html.erb

```diff
@@ -4,6 +4,9 @@
       <div class="user_name_and_location_and_follow">
         <div class="user_name_and_location">
           <h1 class="user_name"><%= @user.name %></h1>
+          <% if authorized? %>
+            <div style="padding: 0.5em 0;"><%= @user.email %></div>
+          <% end %>
```

Comment:
@innatewonderer How do you feel about.moving this into `#user_mod_stats_area` with the rest of the details aimed at moderators?